service/src/main/java/uk/nhs/adaptors/gp2gp/common/constants/Constants.java

```diff
@@ -0,0 +1,6 @@
+package uk.nhs.adaptors.gp2gp.common.constants;
+
+public class Constants {
```

Comment:
"Constants" is not a very descriptive class name. Could I add constant of PI here?

Should this class instead be an enumeration of task names?